srtcore/buffer_rcv.cpp

```diff
@@ -200,6 +200,27 @@ int CRcvBufferNew::dropUpTo(int32_t seqno)
     return iDropCnt;
 }
 
+int CRcvBufferNew::dropAll()
```

Comment:
Why not just call `dropUpTo(m_iStartSeqNo+m_iMaxPosInc)`? So that something likes `m_numOutOfOrderPackets`, `m_iFirstReadableOutOfOrder` would also be updated.

Reply:
OK, updated now.

Reply:
Then we need to know up to what seqno to drop. We can't drop up to the new ISN, because it can be anywhere relative to the current RCV buffer starting position.

Reply:
You are right.

Reply:
Although it might make sense to define the end position within the `dropAll()` function, and then call `dropUpTo(m_iStartSeqNo+m_iMaxPosInc)`, like you suggested. 🤔